oaknational/Oak-Web-Application
# Feat/lesq 301/eyfs journey [LESQ-301]

## Description

- Update to use latest MVs serving EYFS content
- Update subject listing page with alternate heading and paragraph on EYFS
- Update keystage selector to accomodate extra button for EYFS
- make the 'new' tag show dependant on cohort 

## How to test

Not possible to test EYFS until we merge this and publish on the migration environment. However we can test that the site looks the same when not EYFS content is published

1. Go to https://deploy-preview-2232--oak-web-application.netlify.thenational.academy
2. Check the keystage selector does not include EYFS
3. Click a keystage
4. You should not see EYFS in the keystage selector
5. You should not be able to search for EYFS content
6. You should not see any links to any EYFS content

## Screenshots

How it will look when EYFS is published:

<img width=500 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/a2ed59f0-068e-40c1-a7b0-0614c7080085">
<img width=500 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/ababf286-7e4e-4e69-84bf-7c53672f2bdc">
<img width=500 src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/3a404336-fbd6-48d5-a0b6-31cb4b8a1a1c">
